build.yml

```diff
@@ -47,12 +52,10 @@ jobs:
       - ${{ if contains(parameters.image, 'windows') }}:
         - bash: |
             echo "##vso[task.prependpath]$CONDA\Scripts"
-            echo "##vso[task.setvariable variable=scipp_install_prefix]$CONDA\scipp_install"
```

Comment:
Can we avoid this bit entirely be making a cmake preset defining the install dir?

Reply:
The pipeline needs to know the location to be able to set the `PYTHONPATH` further down, so i'm not sure how you would accommodate that?

Reply:
I guess, unless you want to hard-code it to `install` within the repo?

Reply:
Maybe the solution would be to change our cmake setup so Python tests can be run via cmake? Basically CI might just do:

```sh
cmake --preset <preset>
cmake --build --preset <preset>
cmake --test --preset <preset>
```

where `<preset>` can be substituted from Azure with the correct OS?